app/main.py

```diff
@@ -1 +1,48 @@
-# write your code here
+class SoftwareEngineer:
+    def __init__(self, name: str) -> None:
+        self.name = name
+        self.skills = []
+
+    def learn_skill(self, skill: str) -> None:
+        self.skills.extend([skill])
```

Comment:
You are adding one skill here. Using of extend method and transforming skill into list is not needed here. 
```suggestion
    def learn_skill(self, skill: str) -> None:
        self.skills.append(skill)
```